feat: update readme (#1)
eve-autonomy/delivery_reservation_lamp_manager: README.md

> # delivery_reservation_lamp_manager

Reviewer: ```suggestion
# Lamp manager for delivery reservation
```

## Overview

> By changing the lighting pattern of the lamp, this node notifies senders of the on-demand delivery reservation status. 

Reviewer: ```suggestion
By changing lighting pattern of the lamp, this node notifies surrounding workers of on-demand delivery reservation status through `/dio_ros_driver`. 
```

> The lighting pattern is defined as follows.

Reviewer: ```suggestion
Lighting patterns are listed below.
```

<table>
  <thead>
    <tr>
      <th scope="col">Reservation status</th>
	    <th scope="col">lighting pattern</th>
	  </tr>
  </thead>
  <tbody>
    <tr>
      <td>Not reserved</td>

>       <td>Keeps going off</td>

Reviewer: ```suggestion
      <td>Off</td>
```

    </tr>
    <tr>
      <td>In progress</td>
      <td>Blinks</td>
    </tr>
    <tr>
      <td>Reserved</td>

>       <td>Keeps lighting</td>

Reviewer: ```suggestion
      <td>Light up</td>
```

## Input and Output
- input
  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)

>     - `/autoware_state_machine/lock_state` : Reservation status for on-demand delivery.

Reviewer: Add a description about values true/false, 1/0, against to the input from the /autoware_state_machine node.

- output
  - to [dio_ros_driver](https://github.com/tier4/dio_ros_driver)

>     - `/dio/dout3` : GPIO output topic. (this topic remapping from /delivery_reservation_lamp_out)

Reviewer: ```suggestion
    - `/dio/dout3` : Digital-out assignment to a 3rd pin in [0-7] general-purpose outputs. This topic is remapped from `/delivery_reservation_lamp_out`.
```

And, add a description about values true/false, 1/0, against to the output from the /dio_ros_driver node.
